translate-date.yaml

```diff
@@ -4,3 +4,4 @@ formats:
   de: 'd.m.Y H:i'
   en: 'm/d/Y h:iA'
   lt: 'Y-m-d H:i'
+  cs: 'd.m.Y H:i'
```

Comment:
**Chore:** Same here 🙂 Let's keep in alphabetic order

Reply:
Done :)